src/config.cpp

```diff
@@ -117,13 +117,29 @@ Config& Config::parse(int argc, char** argv) {
                 cxxopts::value<std::string>()->default_value("CPU"),
                 "TARGET_DEVICE")
             ("cpu_extension",
-                "a path to shared library containing custom CPU layer implementation. Default: empty.",
+                "A path to shared library containing custom CPU layer implementation. Default: empty.",
                 cxxopts::value<std::string>()->default_value(""),
                 "CPU_EXTENSION")
             ("plugin_config",
-                "a dictionary of plugin configuration keys and their values, eg \"{\\\"CPU_THROUGHPUT_STREAMS\\\": \\\"1\\\"}\". Default throughput streams for CPU and GPU are calculated by OpenVINO",
+                "A dictionary of plugin configuration keys and their values, eg \"{\\\"CPU_THROUGHPUT_STREAMS\\\": \\\"1\\\"}\". Default throughput streams for CPU and GPU are calculated by OpenVINO",
                 cxxopts::value<std::string>(),
-                "PLUGIN_CONFIG");
+                "PLUGIN_CONFIG")
+            ("stateful",
+                "Flag indicating model is stateful",
+                cxxopts::value<bool>()->default_value("false"),
+                "STATEFUL")
+            ("stateful_timeout",
```

Comment:
Perhaps `sequence_timeout` would be better name since we won't time out model but it's sequences?